pkg/stub/handler.go

```diff
@@ -887,12 +932,23 @@ func (h *Handler) createSamples(cfg *v1.Config, updateIfPresent, registryChanged
 
 		err = h.upsertTemplate(template, t, cfg)
 		if err != nil {
+			if kerrors.IsAlreadyExists(err) {
+				continue
+			}
 			return err
 		}
 	}
 	return nil
 }
 
+func (h *Handler) refetchCfgMinimizeConflicts(cfg *v1.Config) *v1.Config {
```

Comment:
I'd add a godoc comment here, since errors here are swallowed.

Reply:
part of next push